Cargo.toml

```diff
@@ -37,41 +37,48 @@ base64 = "0.21"
 bitflags = "2.4"
 chrono = "0.4"
 clap = { version = "4.4", default-features = false, features = [
-  "help",
-  "env",
-  "std",
-  "usage",
-  "error-context",
-  "derive",
-  "cargo",
+    "help",
+    "env",
+    "std",
+    "usage",
+    "error-context",
+    "derive",
+    "cargo",
 ] }
 cloudproof = { version = "2.3", features = ["findex-redis"] }
+const-oid = { version = "0.9", features = ["db"] }
+der = { version = "0.7", features = ["pem"] }
+env_logger = "0.10"
 hex = "0.4"
 http = "0.2"
 josekit = "0.8.3"
 native-tls = "0.2"
+num-bigint = { version = "0.4", features = ["rand", "serde"] }
 num-bigint-dig = { version = "0.8", default-features = false, features = [
```

Comment:
we shouldn't be using `num-bigint` anymore right? keep only `num-bigint-dig` instead?